modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-storage-discovery_{context}"]
+= Automating discovery and provisioning for local storage devices
+
+The Local Storage Operator automates local storage discovery and provisioning. With this feature, you can simplify installation when dynamic provisioning is not otherwise available during deployment, such as with bare metal, VMware, or AWS store instances with attached devices.
+
+[IMPORTANT]
+====
+Automatic discovery and provisioning is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Use the following procedure to create discovery requests, automatically discover local devices, and automatically provision local volumes for selected devices.
```

Comment:
I am wondering if this might be more scannable for readers if it is a bulleted list. For example:

"You can use this procedure to complete the following tasks:

* Create discovery requests
* Automatically discover local devices
* Automatically provision local volumes for selected devices"

Reply:
I do like the list better, but I ended up removing one of the steps, so it's really down to just two items now, so I'll keep as a sentence. :)